src/views/Dashboard/CustomerDashboard.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-unused-vars */
+
+import React, { Component, Fragment, Suspense } from 'react';
+import { Container, Row, Col } from 'reactstrap';
+import routes from '../../route';
+import Navigation from '../../_nav';
+import AppHeader from '../../hoc/AppHeader';
+import AppSidebar from '../../hoc/AppSidebar';
+
+export default class CustomerDashboard extends Component {
+  loading = () => <div>Loading...</div>;
+
+  render() {
+    return (
+      <Fragment>
+        <Container>
+          <Row>
+            <Col>
+              <Suspense>
+                <AppHeader />
+              </Suspense>
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Suspense>
+                <AppSidebar navConfig={Navigation} {...this.props} />
+              </Suspense>
+            </Col>
+            <Col>
+              <Container fluid>
+                <Suspense fallback={this.loading()}>
+                  <Switch>
+                    {routes.map((route, idx) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)